### ISSUE TITLE
AWS `span.resource` does not give a meaningful insight on service page

### ISSUE THREAD

**Issue by jfrancoist** [DataDog/dd-trace-rb#374]
When using the current `:aws` instrumentation. We get the following resource

![image](https://user-images.githubusercontent.com/6594266/37643632-798c28ae-2c21-11e8-8ad1-63e8e35aa0ba.png)

I think it would make more sense to have the name of the AWS command listed here instead. Which would result finding out which command is performing poorly instead of having to go through each trace on the next screen?

![image](https://user-images.githubusercontent.com/6594266/37643836-1a9d80c6-2c22-11e8-86ec-0836745f6e99.png)

This line: https://github.com/DataDog/dd-trace-rb/blob/master/lib/ddtrace/contrib/aws/instrumentation.rb#L31 could change to

```ruby
span.resource = context.safely(:resource)
```

making the service page show a list of AWS command in the format [aws_service].[method] e.g **s3.head_object**

And maybe https://github.com/DataDog/dd-trace-rb/blob/master/lib/ddtrace/contrib/aws/instrumentation.rb#L30 to 

```ruby
span.name = RESOURCE
```

**Comment by delner** [comment]
From a quick 2 minute look, this seems reasonable and could address your issue.

May I suggest opening a pull request and showing the effect it has through a couple more screenshots? Would probably be the best means of having this expedited!

**Comment by jfrancoist** [comment]
Hi David,

I took the opportunity to create a small PR and with some screenshots of
the resulting behaviour.

JF

**Comment by delner** [comment]
Awesome, thanks a lot. I'll take a look!

**Comment by delner** [comment]
This should be fixed in #377 and will roll with `0.12.1`, closing this issue.
